### ISSUE TITLE
Allow screen readers to jump to main content

### ISSUE THREAD

**Issue by 6a68** [mozilla-services/screenshots#4666]
Notes:

- replace structural divs with semantic elements like header, footer, nav, main, article, section
- can also use ARIA landmark roles, though these largely duplicate semantic element info (for instance, role="main", role="navigation")
- each page should have an h1 for the most important header

**Comment by 6a68** [comment]
Related aXe error: "page must contain one main landmark"

**Comment by punamdahiya** [comment]
Should be picked with UI fixes in #4686, #4685. 
@6a68  it will be helpful to add steps here on how to reproduce and test fix of this issue with screen reader.

**Comment by punamdahiya** [comment]
Closing issue as duplicate of #5016
